<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordChangeDialog</class>
 <widget class="QDialog" name="PasswordChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Old password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Repeat new password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oldp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>128</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>128</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rnewp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>128</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushOk">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
